src/lib/translations.json

```diff
@@ -30,6 +30,10 @@
       "key": "styleguide/CommentComposer/wait",
       "value": "Sie können {time} wieder einen Beitrag schreiben."
     },
+    {
+      "key": "styleguide/CommentComposer/formattingHelp",
+      "value": "Benutzen Sie ein Backslash vor dem Gendersternchen damit es nicht mit kursiv verwechselt wird:\nLeser\\*in"
```

Comment:
Hinweis funktioniert nicht, weil die Erklärung fehlt, warum das mit kursiv verwechselt werden würde. Habe das Gefühl, das stiftet Verwirrung. (Auf der [Formatier-Seite](https://www.republik.ch/markdown) klappts, weil die Kursivmachung gleich voran steht.)

Vielleicht eher generell: «Formatieren Sie Ihren Beitrag mit Sonderzeichen (Link Formatier-Seite)»

Vielleicht nur Anweisung: «Damit System Gendersternchen erkennt, stellen Sie einen Backslash voran: Leser\\*in»

Vielleicht ganz kurz: «Gendersternchen setzen: \\*»

Reply:
Hab den Hinweis von der /markdown Seite übernommen. Ich find alle Lösungen ok

Reply:
Hab "Damit System Gendersternchen erkennt, stellen Sie einen Backslash voran: Leser\*in" übernommen